src/app/utils/yamlDump.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line no-restricted-imports
+import { dump } from 'js-yaml'
+
+export function yamlDump(obj: any) {
+  return dump(obj, {
+    replacer: (k, v) => {
+      return v
```

Comment:
Why don't we need the keys ?

Reply:
Key can be used to change the value return format, but it isn't itself expected to be returned. This replacer is noop - until next PR